GoogleDataTransport/GDTCCTLibrary/GDTCCTUploader.m

```diff
@@ -171,30 +159,40 @@ - (nullable NSURL *)serverURLForTarget:(GDTCORTarget)target {
     CSHServerURL = [NSURL URLWithString:[NSString stringWithUTF8String:URL]];
   });
 
+  return @{
```

Comment:
Can we create a static local dictionary and create the mapping just once and return that value everytime we call?With the current implementation, we would be creating a dictionary every time we call this API.

Reply:
Sounds good! Fixed.